bin/install-qa-check.d/90cython-dep

```diff
@@ -0,0 +1,41 @@
+# Check for missing dev-python/cython dep based on output
+# from setuptools cythonize() support
+
+cython_dep_check() {
+	[[ ${BDEPEND} == *dev-python/cython* ]] && return
```

Comment:
```suggestion
	[[ ${BDEPEND}${DEPEND} == *dev-python/cython* ]] && return
```

Reply:
This check doesn't look at whether distutils-r1 is inherited, though.

Reply:
Do we really care more about a potential false positive of an ebuild using ancient EAPI, or that people put it wrongly in `DEPEND` instead of `BDEPEND`?

Reply:
No. :)